$OpenBSD: patch-programs_Makefile,v 1.5 2017/01/05 22:04:37 sthen Exp $

Don't build -m32 variants.
Don't build fuzzer/frametest (requires patches for ftime).

--- programs/Makefile.orig	Tue Nov 22 20:34:17 2016
+++ programs/Makefile	Tue Jan  3 21:49:21 2017
@@ -83,7 +83,7 @@ clean:
 #------------------------------------------------------------------------
 #make install is validated only for Linux, OSX, kFreeBSD, Hurd and
 #FreeBSD targets
-ifneq (,$(filter $(shell uname),Linux Darwin GNU/kFreeBSD GNU FreeBSD))
+ifneq (,$(filter $(shell uname),Linux Darwin GNU/kFreeBSD GNU FreeBSD OpenBSD))
 
 install: lz4$(EXT) lz4c$(EXT)
 	@echo Installing binaries
